test/test.clients.ts

```diff
@@ -131,10 +133,10 @@ describe('Clients', function () {
   it('should support default params', function (done) {
     var google = new googleapis.GoogleApis();
     var datastore = google.datastore({
-      version: 'v1beta2',
+      version: 'v1beta3',
```

Comment:
Funny story!  The v1beta2 release of the datastore *doesn't exist anymore*.  These unit tests are currently failing in the master branch.  I switched to `v1beta3` which actually changed the API surface.  So ...  I had to switch the unit tests.